cmd/falco/runner.go

```diff
@@ -51,6 +51,12 @@ type StatsResult struct {
 	Lines       int    `json:"lines"`
 }
 
+type Fetcher interface {
+	Backends() ([]*remote.Backend, error)
```

Comment:
I thought about this as well and I went for the lazy way since I wouldn't do anything besides reusing the same abstraction pretty much but long term we should have our own internal abstraction rather than reusing what fastly provides.

Will change that

Reply:
fixed in 69679c7bb5276feb8b561e2faa2af3b69794b4c1